torch/distributed/distributed_c10d.py

```diff
@@ -2382,6 +2386,8 @@ def barrier(group=GroupMember.WORLD,
         group (ProcessGroup, optional): The process group to work on. If None,
             the default process group will be used.
         async_op (bool, optional): Whether this op should be an async op
+        device_ids ([int], optional): List of device/GPU ids
```

Comment:
nit: Need a comma or fullstop after `ids` and before `Valid`

Reply:
Done